src/components/study-container.js

```diff
@@ -317,20 +322,18 @@ export function StudyContainer({ view, onChangeTab }) {
                         });
                     });
 
-                let firstBuiltNode = getFirstNodeOfType(
-                    tree,
-                    'NETWORK_MODIFICATION',
-                    'BUILT'
-                );
-
-                dispatch(
-                    setCurrentTreeNode(
-                        firstBuiltNode
-                            ? firstBuiltNode
-                            : getFirstNodeOfType(tree, 'ROOT')
-                    )
-                );
-
+                let firstSelectedNode =
+                    getFirstNodeOfType(tree, 'NETWORK_MODIFICATION', 'BUILT') ||
+                    getFirstNodeOfType(tree, 'ROOT');
+
+                // To get positions we must get the node from the model class
+                let ModelFirstSelectedNode = {
```

Comment:
Why let and not const ? (can not remember a language where constness cost less typing than mutability, though :-/ )